person.rb

```diff
@@ -0,0 +1,38 @@
+# creating a Person class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1000..9999)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  # define getter for @name
+  attr_accessor :name
+
+  # define setter for @name
+
+  # define getter for @age
+  attr_accessor :age
```

Comment:
[OPTIONAL] You could define accessor in only one line, like `attr_accessor :name, :age`